### ISSUE TITLE
Isn't there any 3rd party or community forum for hexya yet

### ISSUE THREAD

**Issue by 1AbdulkarimMousa** [hexya-erp/hexya#135]
I'd like to see this project growing

**Comment by mungaij83** [comment]
I also would like to see an active community in the project. Even if its a developer group on google groups.

**Comment by gbtechnology** [comment]
@mungaij83 I think main reason why the project is on hold is the lack of community support. Maybe we can be more pro-active on this aspect :)

**Comment by mungaij83** [comment]
Hi @gbtechnology , 

Let's create slack developer team or google group for this project. I am experienced with odoo and python in general and would like to contribute to this project during my free time. We can always attract the community with community interaction. 

Specifically, we can have the following channels in the project(Slack or Groups):
1. Customer or Q&A channel
2. Developer Channel
3. Feature tracking channel

:)

**Comment by silverf0x0** [comment]
Hi,

This Is fascinating. I'll open slack channel and I'll post details here :)
Best,
Globalist Technology (Raffaele - silverf0x0)

Il ven 25 nov 2022, 15:06 Njoroge John M. ***@***.***> ha
scritto:

**Comment by 1AbdulkarimMousa** [comment]
Hi silverf0x0, kindly post it here

**Comment by gbtechnology** [comment]
Hi,

to all people interested to join the new slack channel, here it is:
https://join.slack.com/t/nuovaareadila-3id2004/shared_invite/zt-1kj6ncmas-TI4DTVaVvHfg1CFCoAm15Q

Il giorno ven 25 nov 2022 alle ore 19:29 1AbdulKareem <
***@***.***> ha scritto:

**Comment by mungaij83** [comment]
Joined.. 

Kindly update the same on readme for easy discovery by new community members who don't have knowledge of this conversation. Maybe under "how to contribute" section.

**Comment by npiganeau** [comment]
You can submit a PR for README modification if you like.

**Comment by mungaij83** [comment]
Hi @npiganeau, 

I have modified the README.md and wanted to push a pull request. Which model will be followed since branch PR does not work. Gives error 403.  Do I need to fork the repository first?

**Comment by dharmjit** [comment]
The slack link is not active anymore. Also the demo link on the website is not working.

**Comment by gbtechnology** [comment]
Hi @dharmjit I confirm that the slack link is https://hexya.slack.com.
If you have troubles, you can send me your mail at globalist.technology.com and I'll add it within the workplace.

Best,
Raffaele

**Comment by kiseeb** [comment]
> Hi @dharmjit I confirm that the slack link is https://hexya.slack.com. If you have troubles, you can send me your mail at globalist.technology.com and I'll add it within the workplace.
> 
> Best, Raffaele



@gbtechnology @mungaij83 

Kindly add my email to the workspace

abdulkarim.mousa@outlook.com

**Comment by fakhrulraharjo** [comment]
> Hi @dharmjit I confirm that the slack link is https://hexya.slack.com.
> If you have troubles, you can send me your mail at globalist.technology.com and I'll add it within the workplace.
> 
> Best,
> Raffaele
> 

Hi @gbtechnology i really interested to help contributing to this project. Please add me to workspace at fakhrulraharjo@gmail.com

Regards

**Comment by jhones727** [comment]
Yes, there are third-party forums for Hexya discussions. You can check platforms like GitHub Discussions or community forums related to Hexya. These spaces often provide a wealth of information and support from the Hexya community. Happy exploring [visit](https://globalfist.com/)!

**Comment by juliaangel78** [comment]
You might find discussions and support for Hexya on platforms like GitHub or Stack Overflow. They're often good places for community-driven help and third-party resources you can check the [site](https://minidoodlepets.com/).

**Comment by gbtechnology** [comment]
There's actually also a slack channel we opened about 1 year ago. The
founder is also in the channel → https://app.slack.com/

**Comment by silverf0x0** [comment]
Right slack link area is: https://hexya.slack.com

Il giorno mar 16 lug 2024 alle ore 06:21 Globalist Technology - Raffaele
Del Gatto ***@***.***> ha scritto:

> There's actually also a slack channel we opened about 1 year ago. The
> founder is also in the channel → https://app.slack.com/
>

**Comment by georgemartin72** [comment]
I haven't seen a dedicated community forum for Hexya yet. It might be worth checking broader developer forums or GitHub for related [Show now ](https://armchampionshipbelts.com/) discussions.

**Comment by jacobmark51** [comment]
There's been talk of a community forum in the works [IDM Patch](https://idmcrack.pk/). Keep an eye on updates from the developers.

**Comment by johncrista239** [comment]
If you're looking for more Hexya community support, you might want to check out platforms like Reddit or GitHub Discussions. These can be great places to find or build a community around Hexya [IDM Patch](https://idmcrack.pk/). If nothing's available yet, maybe starting a new forum could help connect others interested in Hexya.

**Comment by Rots** [comment]
Is the slack still used? The invite link above has expired. what about the Gitter room? Can this not be used? https://app.gitter.im/#/room/#hexya-erp_community:gitter.im
